README_CN.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
```

Comment:
It is more conventional to name it README.cn.md?

Reply:
sure